frontend/src/scenes/actions/ActionsTable.tsx

```diff
@@ -112,6 +112,9 @@ export function ActionsTable(): JSX.Element {
         {
             title: '',
             render: function RenderActions(action: ActionType) {
+                const encodedName = encodeURIComponent(action.name)
+                const actionsLink = `/insights?insight=TRENDS&interval=day&display=ActionsLineGraph&actions=%5B%7B%22id%22%3A${action.id}%2C%22name%22%3A%22${encodedName}%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%5D&properties=#backTo=Actions&backToURL=${window.location.pathname}`
```

Comment:
this link seems a little hardcoded with the URIs, perhaps we could break it up into parts for readability?

Reply:
good call. I was mirroring the events table but I just changed both to use a param builder function now